tasks/yarn.yml

```diff
@@ -4,7 +4,7 @@
   git:
     repo: https://github.com/openwisp/openwisp-wifi-login-pages.git
     dest: "{{ wifi_login_pages_path }}"
-    version: master
+    version: 1.1.0
```

Comment:
Can you please add a variable for this and also the repo? So we can change it if needed, eg: `wifi_login_pages_repo_url` and `wifi_login_pages_repo_revision`.